doc/rest-api.md

```diff
@@ -3315,6 +3316,23 @@ Output:
 }
 ```
 
+### `/1.0/storage-pools/<pool>/volumes/<type>/<name>/state`
+#### GET
+ * Description: Storage volume state information
```

Comment:
@stgraber should this section have an `Introduced: with API extension blah` line like the other sections?